src/aind_data_access_api/utils.py

```diff
@@ -0,0 +1,287 @@
+"""Package for common methods used for interfacing with DocDB."""
+
+import re
+from typing import Dict, Iterator, List, Optional
+from urllib.parse import urlparse
+
+from aind_data_schema.core.data_description import DataRegex
+from pymongo import MongoClient
+
+
+def is_prefix_valid(prefix: str) -> bool:
```

Comment:
If this the only function that is requires aind-data-schema, it might be better to remove it and the aind-data-schema dependency for the time being. At the very least, since this package will probably be imported alongside other packages, it will be problematic if the version of aind-data-schema is pinned to a specific version.

Reply:
`is_record_location_valid` uses this function to check that the location field contains a valid data asset name. Should I also remove that check from `is_record_location_valid` for the time being?

Reply:
yeah, let's do those in a separate PR. Hopefully, people shouldn't be modifying the location.

Reply:
Great, I removed those two functions.